packages/grafana-ui/src/components/Table/BarGaugeCell.tsx

```diff
@@ -40,10 +39,21 @@ export const BarGaugeCell: FC<ReactTableCellProps> = props => {
     barGaugeMode = BarGaugeDisplayMode.Lcd;
   }
 
+  let width;
+  if (column.width) {
+    if (typeof column.width === 'string') {
+      width = parseFloat(column.width) - tableStyles.cellPadding * 2;
```

Comment:
if width is "100%" what should we do?  does that just go to 100?  

If we are not really allowing strings, can we force numbers?

Reply:
Gah, yes. The %-case. In other components we're allowing px, % and number, so I don't see why we shouldn't allow them here? Just need to refine the logic I suppose.

Reply:
We agreed to go with numbers(px) only here for now.